CRM/Utils/System/DrupalBase.php

```diff
@@ -660,7 +660,7 @@ public function isFrontEndPage() {
 
     // Get the menu for above URL.
     $item = CRM_Core_Menu::get($path);
-    return !empty($item['is_public']);
+    return (empty($item) || !empty($item['is_public']));
```

Comment:
@swastikpareek Can you add comments here explaining why (use the comments you already made on dev/core#1987)

Reply:
Sure will do.